processing/data_collection/gazette/spiders/rn_macaiba.py

```diff
@@ -0,0 +1,76 @@
+import re
+
+import dateparser
+import datetime as dt
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RnMacaiba(BaseGazetteSpider):
+    start_date: dt.date
+    end_date: dt.date
+
+    TERRITORY_ID = "2407104"
+    EXTRA_ISSUE_PATTERN = r"Edição Extraordinária"
+    name = "rn_macaiba"
+    allowed_domains = ["macaiba.rn.gov.br"]
+
+    def __init__(self, start_date=None, end_date=None):
+        self.start_date = dt.date.today().replace(day=1)
+        self.end_date = dt.date.today()
+
+        if start_date is not None:
+            parsed_data = dateparser.parse(start_date)
+            if parsed_data is not None:
+                self.start_date = parsed_data.date()
+
+        if end_date is not None:
+            parsed_data = dateparser.parse(end_date)
+            if parsed_data is not None:
+                self.end_date = parsed_data.date()
+
+        self.logger.debug(
+            "Start date is {date}".format(date=self.start_date.isoformat())
+        )
+        self.logger.debug("End date is {date}".format(date=self.end_date.isoformat()))
+
+    def start_requests(self):
+        yield scrapy.FormRequest(
+            "https://www.macaiba.rn.gov.br/boletins/busca",
+            method="POST",
+            formdata={
+                "inicio": self.start_date.isoformat(),
+                "fim": self.end_date.isoformat(),
+            },
+        )
+
+    def parse(self, response):
+        for item in response.css("div.conteudo li.collection-item div"):
+            gazette = item.css("::text").get().strip()
```

Comment:
Maybe you can `item.css("::text").re(regex here)`, not sure in this case

Reply:
I can't use `re.IGNORECASE` on `SelectorList.re`